### PR TITLE
expression: implement vectorized evaluation for `builtinNameConstJSONSig`

### PR DESCRIPTION
**Pull request by Baytwt**
### What problem does this PR solve? 
expression: implement vectorized evaluation for `builtinNameConstJSONSig`
### What is changed and how it works?
expression: implement vectorized evaluation for `builtinNameConstJSONSig`

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by qw4990** [comment]
Please post the benchmark result in your PR's description~

File: expression/builtin_miscellaneous_vec_test.go [hunk 1]
```diff
@@ -80,6 +80,7 @@ var vecBuiltinMiscellaneousCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETDecimal, childrenTypes: []types.EvalType{types.ETString, types.ETDecimal}},
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETInt}},
 		{retEvalType: types.ETReal, childrenTypes: []types.EvalType{types.ETString, types.ETReal}},
+		{retEvalType: types.ETJSON, childrenTypes: []types.EvalType{types.ETString, types.ETJSON}},
```
Comment (qw4990) [inline expression/builtin_miscellaneous_vec_test.go]: `types.ETJSON` -> `types.ETJson`.

Comment (qw4990) [reply expression/builtin_miscellaneous_vec_test.go]: @Baytwt Friendly ping, please fix this problem and sign our CLA license.

Comment (Baytwt) [reply expression/builtin_miscellaneous_vec_test.go]: fixed

**Comment by qw4990** [comment]
@Baytwt Friendly ping, please sign the CLA license :D

**Pull request merged**